aws/resource_aws_db_instance.go

```diff
@@ -970,6 +1013,9 @@ func resourceAwsDbInstanceRead(d *schema.ResourceData, meta interface{}) error {
 	d.Set("kms_key_id", v.KmsKeyId)
 	d.Set("port", v.DbInstancePort)
 	d.Set("iam_database_authentication_enabled", v.IAMDatabaseAuthenticationEnabled)
+	d.Set("performance_insights_enabled", v.PerformanceInsightsEnabled)
+	d.Set("performance_insights_kms_key_id", v.PerformanceInsightsKMSKeyId)
+	d.Set("performance_insights_retention_period", v.PerformanceInsightsRetentionPeriod)
```

Comment:
We might need to convert this attribute's type before `d.Set()` -- this can be verified with either:

```go
if err := d.Set("performance_insights_retention_period", v.PerformanceInsightsRetentionPeriod); err != nil {
  return fmt.Errorf("error setting performance_insights_retention_period: %s", err)
}
```

Or running the acceptance testing with `TF_SCHEMA_PANIC_ON_ERROR=1`.

If it does have an error (like cannot convert `*int64` to `int`), then the fix is to use: `int(aws.Int64Value(v.PerformanceInsightsRetentionPeriod))`